gensrc/thrift/PaloInternalService.thrift

```diff
@@ -77,6 +77,10 @@ struct TLoadErrorHubInfo {
     3: optional TBrokerErrorHubInfo broker_info;
 }
 
+struct TResourceLimit {
+    1: optional i32 cpu_limit
```

Comment:
Maybe It is better to use a name such as scannode_max_parallelism xxx

Reply:
I want to decouple the struct with the certain implementation.
So that we can extend this feature in future.

Reply:
maybe use max_concurrency is better

Reply:
> maybe use max_concurrency is better

In fact, what I want to express is that this value is not bound to any implementation. It does not represent the number of threads or the number of concurrency, but a relative value.
It is more like a priority concept, or similar to the concept of weight in cgroups.
In this way, we can continue to use this parameter when we implement more refined resource isolation in the future.